crates/ethereum-engine/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "ethereum-engine"
```

Comment:
This package name should probably be somewhat more specific. Maybe `ethereum-settlement-engine` or `ilp-settlement-ethereum` if we want to be really specific